workitem/list_type_blackbox_test.go

```diff
@@ -184,3 +184,67 @@ func TestListType_SetDefaultValue(t *testing.T) {
 		})
 	}
 }
+
+func TestListType_ConvertToString(t *testing.T) {
```

Comment:
You only test list of strings here but not list of users, ints, floats etc.

Reply:
Done. 3860e94.